tests/FakeItEasy.Tests/Expressions/ExpressionArgumentConstraintFactoryTests.cs

```diff
@@ -186,7 +188,7 @@ public void OnAfterCallIntercepted(ICompletedFakeObjectCall call, IFakeObjectCal
 
                 if (call.Method.Name.Equals("TrapConstraints"))
                 {
-                    call.GetArgument<Action>(0).Invoke();

```

Comment:
This was causing the constraint to be executed outside the trap; I just used a real trap instead.